markhor_bringup/launch/camera_rearview.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<launch>
+    
+    <!-- MS Lifecam Cinema Camera -->
+    <node name="camera_rearview" pkg="usb_cam" type="usb_cam_node" output="screen">
+    <param name="video_device" value="/dev/video0" />
+    <param name="image_width" value="640" />
```

Comment:
I don't think these are the specs of our camera

Reply:
Could you show me the documentation you use to decide the parameters for the launch file ?

Reply:
Used the file from takin. Guess I'll removed camera_rearview.launch from the commit.